index.bs

```diff
@@ -353,37 +368,43 @@ to the track satisfies the track's constraints. Each track may receive a
 </p>
 
-The <dfn>closeWritable</dfn> algorithm is given a |generator| as input.
+The <dfn>closeWritable</dfn> algorithm is given a |source| as input.
 It is defined by running the following steps.
-1. For each track `t` connected to |generator|, {{EndTrack|end}} `t`.
+1. For each track `t` sourced from |source|, {{EndTrack|end}} `t`.
 2. Return [=a promise resolved with=] undefined.
 
+</dd>
+<dt><dfn attribute for=VideoTrackSource>muted</dfn></dt>
+<dd>Mutes the {{VideoTrackSource}}. The getter steps are to return
+[=this=].`[[isMuted]]`. The setter steps, given a value |newValue|, are as follows:
+1. If |newValue| is equal to [=this=].`[[isMuted]]`, abort these steps.
+1. Set [=this=].`[[isMuted]]` to |newValue|.
+1. Unless one has been queued already this run of the event loop, [=queue a task=] to run the following steps:
+    1. Let |settledValue| be [=this=].`[[isMuted]]`.
+    1. For each live track sourced by [=this=], [=queue a task=] to [=set a track's muted state=] to |settledValue|.
+
+</dd>
+<dt><dfn attribute for=VideoTrackSource>track</dfn></dt>
+<dd>The {{MediaStreamTrack}} output. The getter steps are to return
+[=this=].`[[track]]`.
 </dd>
 </dl>
 
-### Specialization of MediaStreamTrack behavior ### {#generator-as-track}
-A {{MediaStreamTrackGenerator}} is a {{MediaStreamTrack}}. This section adds
-clarifications on how a {{MediaStreamTrackGenerator}} behaves as a
-{{MediaStreamTrack}}.
+### Specialization of MediaStreamTrack behavior ### {#video-source-as-track}
+A {{VideoTrackSource}} acts as the source for one or more {{MediaStreamTrack}}s.
+This section adds clarifications on how a {{MediaStreamTrack}} sourced from a
+{{VideoTrackSource}} behaves.
 
-#### clone #### {#generator-clone}
-The {{MediaStreamTrack/clone}} method on a {{MediaStreamTrackGenerator}}
-returns a new {{MediaStreamTrack}} object whose source is the
-same as the one for the {{MediaStreamTrackGenerator}} being cloned.
-This source is the {{MediaStreamTrackGenerator/writable}} field of
-the {{MediaStreamTrackGenerator}}.
+#### stop #### {#video-source-stop}
+The {{MediaStreamTrack/stop}} method stops the track. When the last track
+sourced from a {{VideoTrackSource}} ends, that {{VideoTrackSource}}'s
+{{VideoTrackSource/writable}} is [=WritableStream/closing|closed=].
 
-#### stop #### {#generator-stop}
-The {{MediaStreamTrack/stop}} method on a {{MediaStreamTrackGenerator}} stops
-the track. When the last track connected to
-the {{MediaStreamTrackGenerator/writable}} of a {{MediaStreamTrackGenerator}}
-ends, its {{MediaStreamTrackGenerator/writable}} is [=WritableStream/closing|closed=].
 
 #### Constrainable properties #### {#generator-constrainable-properties}
 
-The following constrainable properties are defined for video
-{{MediaStreamTrackGenerator}}s and any {{MediaStreamTrack}}s sourced from
-a {{MediaStreamTrackGenerator}}:
+The following constrainable properties are defined for any {{MediaStreamTrack}}s sourced from
+a {{VideoTrackGenerator}}:
```

Comment:
VideoTrackGenerator -> VideoTrackSource

Reply:
Applies to other places below

Reply:
Thanks - at some point I unconsciously thought the name was VideoTrackGenerator. I actually like that name better than VideoTrackSource, but let's go with Jan-Ivar's name in this PR.